### PR TITLE
#167039578 Update Readme Page

### PR DESCRIPTION
**Pull request by Manorlds-Eaglespark**
#### What does this PR do?
Updates the Readme.md to be more descriptive.

#### Description of Task to be completed?
A Description of the project in terms of how to set up, and the best way possible to use the application along with side badges and what the whole project is about.

#### How should this be viewed?
Use this [Link](https://github.com/andela/ah-frontend-kronos/tree/ch-update-readme-167039578)

#### What are the relevant pivotal tracker stories?
[#167039578](https://www.pivotaltracker.com/story/show/167039578)

#### Screenshots (if appropriate)
Before
Screenshot 2019-07-03 at 13 59 02
After
Screenshot 2019-07-03 at 13 59 33

### CODE REVIEW

**Review by reifred — CHANGES_REQUESTED** [review]
Thanks @Manorlds-Eaglespark for updating our readMe file however kindly use a single commit.

File: README.md [hunk 1]
```diff
@@ -1 +1,69 @@
...
+* [Usage](#usage)
+* [Testing](#testing)
+* [Demo Link](#demo-link)
+
+<!-- ABOUT THE PROJECT -->
+## About Authors Haven
+
+This is a practice project to learn and practice React library in making of a frontend. Find the backend repo [here](https://github.com/andela/ah-backend-kronos). Authors Haven is a tool for bloggers and readers to share and find their creative written content.
+
+### Built With
```
Comment (Emmanuel-Dominic) [inline README.md:31]: @Manorlds-Eaglespark, Could you try to follow some of the linting conventions of spaces after every title and lists. helpful resource - https://github.com/DavidAnson/vscode-markdownlint#configure
Thank you.
Screenshot 2019-07-03 at 15 04 47

**Comment by Rhytah** [comment]
> Thanks @Manorlds-Eaglespark for updating our readMe file however kindly use a single commit.

@Manorlds-Eaglespark just as @reifred pointed out let's stick to our PR conventions. In case you need some more insight into squashing commits, here's a helpful resource https://blog.carbonfive.com/2017/08/28/always-squash-and-rebase-your-git-commits/

**Comment by Manorlds-Eaglespark** [comment]
> Thanks @Manorlds-Eaglespark for updating our readMe file however kindly use a single commit.

Thanks, @reifred. This has been resolved.

**Comment by Manorlds-Eaglespark** [comment]
> > Thanks @Manorlds-Eaglespark for updating our readMe file however kindly use a single commit.
> 
> @Manorlds-Eaglespark just as @reifred pointed out let's stick to our PR conventions. In case you need some more insight into squashing commits, here's a helpful resource https://blog.carbonfive.com/2017/08/28/always-squash-and-rebase-your-git-commits/

Thanks, @Rhytah for that resource, it was helpful.

**Review by reifred — APPROVED** [review]
Thanks @Manorlds-Eaglespark for working on the requested changes.

**Review by Rhytah — APPROVED** [review]

**Review by Yiga-fred — APPROVED** [review]

**Pull request merged**